ocamldoc/Makefile

```diff
@@ -194,27 +195,20 @@ dot: ocamldoc.dot
 ocamldoc.dot: $(EXECMOFILES)
 	$(OCAMLDOC_RUN) -dot -dot-reduce -o $@ $(INCLUDES) odoc*.ml
 
-# Parsers and lexers dependencies :
-###################################
-odoc_text_parser.ml: odoc_text_parser.mly
-odoc_text_parser.mli: odoc_text_parser.mly
+# Lexers and parsers
```

Comment:
The parsers and lexers are still defined for the sake of the `.depend` target, would it make sense to keep the previous section title?